core/sqf/sqenvcom.sh

```diff
@@ -95,17 +95,20 @@ cpucnt=$(grep processor /proc/cpuinfo | wc -l)
 #     no number means unlimited, and will swamp the system
 export MAKEFLAGS="-j$cpucnt"
 
+# For now set up the TOOLSDIR, it may be overwritten later when the
+# .trafodion configuration file is loaded.
 if [[ -n "$CLUSTERNAME" ]]; then
-  export MY_ROOT=/opt/hp
-  export TOOLSDIR=${TOOLSDIR:-/home/tools}
+  if [ -z "$TOOLSDIR" ]; then
+    export TOOLSDIR=${TOOLSDIR:-/home/tools}
+  fi
```

Comment:
Setting TOOLSDIR here seems to be doing same thing as before, but more complicated (3 lines instead of one).  If we are explicitly checking for zero-length, then we might as well use the simple assignment:
   export TOOLSDIR=/home/tools